src/main/resources/config.toml

```diff
@@ -68,6 +68,9 @@ allow_addowner_for_offline_players_without_lp = false
 # Whether or not members of a region can /ps home to the region.
 allow_home_teleport_for_members = true
 
+# Whether or not you want to bypass the Worldguard passthrough flag
+bypass_wg_passthrough = true
```

Comment:
I think this should be a setting in the blocks config rather than set globally.

It would be more general in my opinion to call it `placing_bypasses_wg_passthrough` or something along those lines.

Reply:
Yes the block config makes more sense. I changed it , should i do another pull request ?

Reply:
You can simply push new commits to your branch which will update this PR